data/items.json

```diff
@@ -1,70 +1,3 @@
 [
-  {
-    "type": "Movie",
-    "id": 309,
-    "title": "Gladiator",
-    "genre_id": "",
-    "author_id": "",
-    "source_id": 430,
-    "label_id": "",
-    "publish_date": "2020-10-13",
-    "archived": false,
-    "silent": false
-  },
-  {
-    "type": "Movie",
-    "id": 255,
-    "title": "Trip to the moon",
-    "genre_id": "",
-    "author_id": "",
-    "source_id": 430,
-    "label_id": "",
-    "publish_date": "1900-05-24",
-    "archived": false,
-    "silent": true
-  },
-  {
-    "type": "Book",
-    "id": 141,
-    "title": "Once Uppon a Time",
-    "publish_date": "2000-09-18",
-    "publisher": "Marvel",
-    "cover_state": "good",
-    "archived": false,
-    "label_id": 629
-  },
-  {
-    "type": "Book",
-    "id": 817,
-    "title": "The Great Wolf",
-    "publish_date": "2022-03-18",
-    "publisher": "Klaus",
-    "cover_state": "bad",
-    "archived": false,
-    "label_id": 417
-  },
-  {
-    "type": "MusicAlbum",
-    "id": 506,
-    "title": "Music",
-    "genre_id": 448,
-    "author_id": "",
-    "source_id": "",
-    "label_id": "",
-    "publish_date": "2000-09-18",
-    "archived": false,
-    "on_spotify": true
-  },
-  {
-    "type": "MusicAlbum",
-    "id": 707,
-    "title": "Motomami",
-    "genre_id": 830,
-    "author_id": "",
-    "source_id": "",
-    "label_id": "",
-    "publish_date": "2022-03-18",
-    "archived": false,
-    "on_spotify": true
-  }
+  null
```

Comment:
Please make sure that ALL the information in this file is preserved, and that it stays there when the program loads and closes